addons/arsenal/functions/fnc_replaceUniqueItemsLoadout.sqf

```diff
@@ -98,9 +98,10 @@ private _cfgVehicles = configFile >> "CfgVehicles";
 
                             // Check weapon & weapon attachments
                             {
-                                // Magazines
+                                // Magazines in weapons have 2 entries: Name and ammo count
                                 if (_forEachIndex in [4, 5]) then {
-                                    _uniqueBaseCfgText = (getText (_cfgMagazines >> _x >> QGVAR(uniqueBase))) call EFUNC(common,getConfigName);
+                                    _x params ["_magazine"];
+                                    _uniqueBaseCfgText = (getText (_cfgMagazines >> _magazine >> QGVAR(uniqueBase))) call EFUNC(common,getConfigName);
 
                                     if (_uniqueBaseCfgText != "") then {
                                         _weaponsInfo set [_forEachIndex, _uniqueBaseCfgText];
```

Comment:
is it a problem that we lose the ammo count?

should this be something like 
```
_weaponsInfo set [_forEachIndex, [_uniqueBaseCfgText, _x # 1]];
``

Reply:
It still works and refills the magazine. I don't think it's a problem.

Reply:
Have you tested saving a loadout and reloading it?

I'm have a feeling that this fix can't work, as I had to write [this](https://github.com/acemod/ACE3/blob/93520bdc9d9db63852cfc555595bfc855ae7e52a/addons/arsenal/functions/fnc_replaceUniqueItemsLoadout.sqf#L48) for weapons not in containers. `getUnitLoadout` returns weapons in containers in the same format as weapons not in containers, so I presume the code should be the same and I forgot to correct that in #9040.

I'll take a in-depth look tonight.

Reply:
It's as I thought unfortunately.

I fixed it in #9297.